Crucible.GameAPI/CrucibleTraderNpcTemplate.cs

```diff
@@ -47,6 +57,153 @@ internal CrucibleTraderNpcTemplate(NpcTemplate npcTemplate)
         /// </summary>
         public Dictionary<int, TraderSettings> TraderSettings => this.GetTraderSettings();
 
+        /// <summary>
+        /// Gets or sets the chapter at which this npc unlocks.
+        /// </summary>
+        public int UnlockAtChapter
+        {
+            get => this.NpcTemplate.unlockAtChapter;
+            set
+            {
+                if (value < 1 || value > 10)
+                {
+                    throw new ArgumentException("Chapter values must range from 1 to 10. Unable to set UnlockAtChapter");
+                }
+
+                this.NpcTemplate.unlockAtChapter = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the minimum and maximum karma range that this trader will spawn in.
+        /// </summary>
+        public (int, int) KarmaForSpawn
```

Comment:
Another tupple that should be broken out int a max and min prop

Reply:
Added two properties for min/max.